packages/components/src/tabs/src/components/osds-tabs/core/controller.ts

```diff
@@ -54,9 +54,14 @@ class OdsTabsController {
 
   handleArrowKey(event: KeyboardEvent): void {
     const currentSelectedTabIndex = this.tabItems.findIndex((tab) => tab.hasAttribute('active'));
-    if(currentSelectedTabIndex === 0 && event.code === 'ArrowLeft' ) {
+    if (currentSelectedTabIndex === -1 || typeof currentSelectedTabIndex === 'undefined' ) {
       return;
     }
+
+    if(currentSelectedTabIndex === 0 && event.code === 'ArrowLeft') {
+      return;
+    }
+
     if(currentSelectedTabIndex === this.tabItems.length - 1 && event.code === 'ArrowRight' ) {
```

Comment:
**Question**: Accessibility guidelines shows that positive tabindex is best to be avoided. Using multiple positive values for tabIndex confuses screen reader users. Considering these guidelines, why use `this.tabItems.length - 1` ?

Reply:
It's not tabindex as the attribute, but as the index of the array of tabItems